types/react-native-video/index.d.ts

```diff
@@ -1,13 +1,11 @@
-// Type definitions for react-native-video 2.0
+// Type definitions for react-native-video 2.0 and 3.1
```

Comment:
If you are updating the type definitions to match version 3.1, you should only list that version in this line.
If you think that the type definitions for major version 2 may continue to be updated, then you can follow the guidelines [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version) to preserve them.

Reply:
Not really, version 2.x is pretty dead. Will update it thanks!